packages/styles/scss/components/lightbox-media-viewer/_lightbox-media-viewer.scss

```diff
@@ -61,13 +61,6 @@
         max-width: 100%;
         height: auto;
         max-height: 100%;
-
-        @include carbon--breakpoint-down('lg') {
-          position: absolute;
-          margin: auto;
-          top: 0;
-          bottom: 0;
-        }
```

Comment:
hey @caiobozato I remember removing this part too a while back for another bug and I ended up putting it back in because I _think_ (if I remember correctly) the image would otherwise overlay/cover the text content. I tested the deploy preview on browserstack and see that the image is covering the "this is a caption" heading.

<img width="888" alt="Screen Shot 2020-07-08 at 11 54 23 AM" src="https://user-images.githubusercontent.com/54281166/86941526-0bd06400-c112-11ea-8d20-38bb17972f78.png">

Reply:
@annawen1 thanks for sharing that. I noticed this issue is happening even if we don't remove the part of code I mentioned, as we can see in this issue: #2445 Maybe both issues can be fixed in this pull request at once

Reply:
@annawen1 I fixed and tested the issue we mentioned above. But since we already have the #2445 card assigned for it, can we proceed with this PR as is and I will apply the fix in a separated PR connected to card #2445?

Reply:
ah ok, yeah that makes sense to me!